packages/api/tests/models/granules/test-granule-reporting.js

```diff
@@ -235,40 +236,52 @@ test('_validateAndStoreGranuleRecord() does not throw an error for a failing rec
 test('storeGranulesFromCumulusMessage() stores multiple granules from Cumulus message', async (t) => {
   const { granuleModel } = t.context;
 
-  const granule1 = fakeGranuleFactoryV2({
-    files: [fakeFileFactory()]
-  });
-  const granule2 = fakeGranuleFactoryV2({
-    files: [fakeFileFactory()]
-  });
+  const bucket = randomId('bucket-');
+  await S3.createBucket(bucket);
 
-  const cumulusMessage = {
-    cumulus_meta: {
-      execution_name: randomId('execution'),
-      state_machine: 'state-machine',
-      workflow_start_time: Date.now()
-    },
-    meta: {
-      collection: {
-        name: 'name',
-        version: '001'
+  try {
+    const granule1 = fakeGranuleFactoryV2({
+      files: [fakeFileFactory({ bucket })]
+    });
+    const granule2 = fakeGranuleFactoryV2({
+      files: [fakeFileFactory({ bucket })]
+    });
+
+    await Promise.all([
```

Comment:
Was there some mocking going on previously that we're superseding here? I can't see it, but if so, can it be removed?